gwpy/cli/qtransform.py

```diff
@@ -215,7 +215,8 @@ def get_spectrogram(self):
             outseg = Segment(gps, gps).protract(args.plot[self.plot_num])
 
             # use the precomputed ASD as the whitener if needed
-            if self.qxfrm_args.get("whiten"):
+            if type(self.qxfrm_args.get("whiten")) == bool and \
+                    self.qxfrm_args.get("whiten"):
```

Comment:
I'm not a fan of direct type comparisons (neither is PEP-0008), so this might be better constructed as:

```suggestion
            if self.qxfrm_args.get("whiten") is True:
```